### PR TITLE
Add network constants 

### PR DESCRIPTION
**Pull request by drohit-cb**
### What changed? Why?

This PR helps add constants for networks Base Mainnet, Ethereum Holesky, and Ethereum Mainnet.

### CODE REVIEW

File: lib/coinbase/constants.rb [hunk 1]
```diff
@@ -14,6 +14,33 @@ module Coinbase
...
+    display_name: 'Ethereum Holesky',
+    protocol_family: :evm,
+    is_testnet: true,
+    native_asset_id: :eth,
+    chain_id: 17_000
+  )
+
+  ETHEREUM_MAINNET = Network.new(
+    network_id: :ethereum_mainnet,
+    display_name: 'Ethereum',
```
Comment (jazz-cb) [inline lib/coinbase/constants.rb:37]: should be Etherem Mainnet?

Comment (drohit-cb) [reply lib/coinbase/constants.rb]: Agree. Fixed.

**Review by jazz-cb — APPROVED** [review]

File: lib/coinbase/constants.rb [hunk 2]
```diff
@@ -14,6 +14,33 @@ module Coinbase
...
+  BASE_MAINNET = Network.new(
+    network_id: :base_mainnet,
+    display_name: 'Base Mainnet',
+    protocol_family: :evm,
+    is_testnet: false,
+    native_asset_id: :eth,
+    chain_id: 8453
+  )
+
+  ETHERUM_HOLESKY = Network.new(
```
Comment (marcin-cb) [inline lib/coinbase/constants.rb:26]: typo:
```suggestion
  ETHEREUM_HOLESKY = Network.new(
```

**Review by marcin-cb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Pull request closed without merging**